pkg/apis/infinispan/v1/types_util.go

```diff
@@ -525,11 +527,17 @@ func applyLabels(ispn *Infinispan, envvar, annotationName string) error {
 	return err
 }
 
+
 // HasCustomLibraries true if custom libraries are defined
 func (ispn *Infinispan) HasCustomLibraries() bool {
 	return ispn.Spec.Dependencies != nil && ispn.Spec.Dependencies.VolumeClaimName != ""
 }
 
+// HasExternalArtifacts true if external artifacts are defined
+func (ispn *Infinispan) HasExternalArtifacts() bool {
```

Comment:
We should probably change `HasCustomLibraries` to `HasPvcArtifacts` or something similar.

Reply:
Yes, will rename it to the `HasDependenciesVolume`